Lesson13/Kadir/conditionalStatements.js

```diff
@@ -89,14 +172,101 @@ election2(14);
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+let height=parseInt(prompt("Input your height(m)= "));
+let weight=parseInt(prompt("Input your weight(kg)= "));
+
+function indexBMI(h, w){
```

Comment:
Naming is one of the most important things in programming. Avoid using parameter named as a single letter, it's a lot better being explicit and say height and weight (or similar).